main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 			for {
 				prs, resp, _ := client.Search.Issues(
 					// Search query to find PR's
+					context.Background(),
```

Comment:
@shurcooL How can I identify that passing `c` is working as expected? It doesnt give me any errors and starts up my server. Is that enough of an indicator? Honestly I have not really understood what passing the `context` to `go-github` does.